### ISSUE TITLE
Fedora arm-minimal issue: No space left on device

### ISSUE THREAD

**Issue by mcattamoredhat** [virt-s1/rhel-edge#6012]
There is an arm-minimal issue in both Fedora 39 and Fedora Rawhide. During playbook execution, the VM goes unreachable due to `No space left on device.`

```
TASK [reboot to deploy new ostree commit] **************************************
fatal: [192.168.100.51]: UNREACHABLE! => changed=false 
  msg: |-
    mkdir: cannot create directory '/home/admin/.ansible/tmp/ansible-tmp-1702611138.728978-170-108063576427986': No space left on device
  unreachable: true
```

Current failing tests are:
https://github.com/virt-s1/rhel-edge/actions/runs/7211500432/job/19664275422  
https://github.com/virt-s1/rhel-edge/actions/runs/7211502313/job/19664258447

**Comment by mcattamoredhat** [comment]
The VM filesystem seems to be completely populated:

```
[admin@localhost ~]$ df -h
Filesystem      Size  Used Avail Use% Mounted on
/dev/vda3       2.9G  2.8G     0 100% /
devtmpfs        4.0M     0  4.0M   0% /dev
tmpfs           2.0G  100K  2.0G   1% /dev/shm
efivarfs        256K   16K  241K   6% /sys/firmware/efi/efivars
tmpfs           780M  1.0M  779M   1% /run
tmpfs           2.0G   16K  2.0G   1% /tmp
/dev/vda2       459M  293M  137M  69% /boot
/dev/vda1       200M   31M  170M  16% /boot/efi
tmpfs           390M  4.0K  390M   1% /run/user/1000

```

**Comment by henrywang** [comment]
```
TASK [check partition size] ****************************************************
changed: [192.168.100.51] => changed=true 
  cmd:
  - df
  - -h
  delta: '0:00:00.007207'
  end: '2023-12-21 17:30:47.225535'
  msg: ''
  rc: 0
  start: '2023-12-21 17:30:47.218328'
  stderr: ''
  stderr_lines: <omitted>
  stdout: |-
    Filesystem      Size  Used Avail Use% Mounted on
    /dev/vda3       2.9G  2.0G  822M  71% /
    devtmpfs        4.0M     0  4.0M   0% /dev
    tmpfs           1.5G     0  1.5G   0% /dev/shm
    efivarfs        256K   16K  241K   6% /sys/firmware/efi/efivars
    tmpfs           591M  928K  590M   1% /run
    tmpfs           1.5G  108K  1.5G   1% /tmp
    /dev/vda2       459M  293M  137M  69% /boot
    /dev/vda1       200M   31M  170M  16% /boot/efi
    tmpfs           296M  4.0K  296M   1% /run/user/1000
```

**Comment by henrywang** [comment]
Fixed by PR #6582
